### DISCUSSION TITLE
Storing Recipes in IndexedDB

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#26] [Comments]
# Storing Recipes in IndexedDB

A look at storing recipe information in IndexedDB

https://www.raymondcamden.com/2024/06/27/storing-recipes-in-indexeddb

**Comment by fimion** [comment]
Alright, so here is my take on this: https://gist.github.com/fimion/d05eb68b70b57a3f51f5a870de886acc

it ends up using [Set.prototype.intersection](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/intersection)  which works in most modern browsers, but that way you can just iterate over all the things you want.

**Comment by cfjedimaster** [comment]
For reference, another person, Stefan (https://mastodon.social/@stefanfrede), shared a solution as well: https://codepen.io/stefanfrede/pen/wvbbwvY?editors=1111
